app/graphql/types/submission_type.rb

```diff
@@ -36,6 +36,7 @@ class SubmissionType < Types::BaseObject
     nilable_field :location_country_code, String, null: true
     nilable_field :medium, String, null: true
     nilable_field :minimum_price_dollars, Integer, null: true
+    nilable_field :myCollectionArtworkID, String, null: true, method: :my_collection_artwork_id
```

Comment:
To confirm, the capitalization used here (and in `sourceArtworkID` below) is preferred over the style of all the other fields?

Reply:
I think `sourceArtworkID` was explicitly written like that (with ID both capital letters), because otherwise `source_artwork_id` would be compiled as `sourceArtworkId` (d is a lowercase letter). I decided to follow this "convention"